src/main/java/org/carlmontrobotics/robotcode2023/commands/TeleopDrive.java

```diff
@@ -52,34 +55,45 @@ public void execute() {
   }
 
   public double[] getRequestedSpeeds() {
-    double rawForward, rawStrafe, rotateClockwise, deltaT;
-    deltaT = Robot.robot.getPeriod();
     // Sets all values less than or equal to a very small value (determined by the idle joystick state) to zero.
     // Used to make sure that the robot does not try to change its angle unless it is moving,
-    if (Math.abs(fwd.getAsDouble()) <= Constants.OI.JOY_THRESH) rawForward = 0.0;
-    else rawForward = maxForward * fwd.getAsDouble();
-    if (Math.abs(str.getAsDouble()) <= Constants.OI.JOY_THRESH) rawStrafe = 0.0;
-    else rawStrafe = maxStrafe * str.getAsDouble();
-
-    rotateClockwise = rcw.getAsDouble();
+    double forward = fwd.getAsDouble();
```

Comment:
What are the units of `forward`? Are they changing?

Reply:
Not an essential change, but `forward` could be named to reflect what it is e.g. requested next forward velocity(?).

Changing the units/use of the variable is opening an opportunity for bugs.

I don't think there is a bug here... just a note of advice.